packages/plans-grid/src/plans-details/index.tsx

```diff
@@ -169,7 +169,7 @@ const PlansDetails: React.FunctionComponent< Props > = ( { onSelect, locale, bil
 					<tr className="plans-details__feature-row" key="price">
 						<th>
 							{ billingPeriod === 'ANNUALLY'
-								? __( 'Monthly subscription (billed yearly)', __i18n_text_domain__ )
+								? __( 'Monthly Price (billed yearly)', __i18n_text_domain__ )
```

Comment:
"Monthly Price" or "Monthly price"? 🤔

Reply:
That's a good one. On wordpress.com/pricing most of the words are capitalized but on /new we have only _Free domain for One Year_, _Email Support_, _VideoPress Support_ so I'd say they are like this by mistake.

Reply:
My understanding is that we've historically had a mixture/didn't pay attention, but that now we're focusing on moving towards Sentence case and away from Title Case.

Don't take my word for it though, there should be P2s somewhere and dotcom design org folks can maybe confirm in Slack. :-)

Reply:
Yes, there's definitely better readability for Sentence case